src/main/java/frc/robot/commands/cpm/ReportColor.java

```diff
@@ -29,11 +29,11 @@ public void initialize() {
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    SmartDashboard.putString("robot sensor color", cpm.getColor());
+    SmartDashboard.putString("CPM sensor color", cpm.getColor());
```

Comment:
Do we need this command? Can we report all this in the cpm periodic method?

Reply:
Sure thing. Removal of the command will be in the next commit.